addons/portal/static/src/scss/portal.scss

```diff
@@ -524,4 +524,16 @@ form label {
             opacity: 1;
         }
     }
+
+    .o_portal_message_internal_off {
```

Comment:
Meh, but I see it was like that before...

Should be changed to have only the appropriate button on the DOM, but this can wait that we migrate to OWL since this is the kind of changes that will be much easier with it.

Reply:
Copy and pasted from website stuff. But yes, "owl will solve everything".